openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -109,6 +114,33 @@
                             }
                         );
                     }
+
+                    $manageLearnersTab.find('.' + view.SLIDABLE_CLASS).click(
```

Comment:
I guess this is the best way to handle finding classes, but I'm not super happy with it..